### ISSUE TITLE
Hack fix of list view route menus so they accessible on touchevent

### ISSUE THREAD

**Issue by scd** [theCrag/website#1974]
@brendanheywood this has been raised in priority because our paying customer only has touch screens.

I have made a minor adjustment to the UI so that route menu appears if you select the route. There are some second order issues I am working through, but I have confirmed it works in the ipad.

It is actually only a couple of lines of code, but had to move things around so that the mouseover callback was a named function so it could be called elsewhere. 

The way the code currently works is that if we do not detect a touchevent browser then we build the menus on mouse over. 

I have added some code so that the menu is also built if the route is selected. There is a little bit of code to stop it being built twice.

I will push to repo and you can check it out if you want. Otherwise if you know what I am talking about just comment on the general direction.

Unless there is a big problem I am going to do a hot fix tomorrow.

**Comment by brendanheywood** [comment]
I'm not near my dev box, so some things to test:

* what happens when multiple routes are selected?
* what happens on small screens as that dropmenu might get a bit munted

**Comment by scd** [comment]
They are the second order issues I mentioned. 

 - [ ] Rather than updating just the selected menu I need to update all the menus for all selected routes.
 - [x] Tested on an andriod and edit is fine
 - [ ] Consider not showing menu if no permissions (more for small screens as it is a waste of valuable small screen space to have button show which says you cannot press this button)

**Comment by scd** [comment]
There are a couple more issues.

 - [x] Annotations were not selectable. I have made this enhancement and updated the node.select event so that it takes into account different behaviors for a route and an annotation. 
 - [ ] Object stacking is a bit wrong with the menus. I have already spent a bit of time on this and cannot get to the bottom of the issue. I think I will leave this for now, as it is becoming a time sink with marginal benefits.

Will hot fix soon
